src/LearningObjects/LearningObjectRouteHandler.ts

```diff
@@ -168,10 +169,29 @@ export function initializePrivate({
     }
   };
 
+  const getLearningObjectChildren = async (req: Request, res: Response) => {
+    try{
+      const id = req.params.id;
+      const children = await LearningObjectInteractor.getLearningObjectChildrenById(
+        dataStore, 
+        id
+      );
+      res.status(200).send(children); 
```

Comment:
This should use the json method to send children back, so that the outbound encoding is standardized.

```suggestion
      res.status(200).json(children); 
```